commands/monitoring.go

```diff
@@ -325,29 +325,31 @@ func getComparator(compareStr string) (godo.AlertPolicyComp, error) {
 
 func validateAlertPolicyType(t string) error {
 	validAlertPolicyTypes := map[string]struct{}{
-		godo.DropletCPUUtilizationPercent:                  {},
-		godo.DropletMemoryUtilizationPercent:               {},
-		godo.DropletDiskUtilizationPercent:                 {},
-		godo.DropletDiskReadRate:                           {},
-		godo.DropletDiskWriteRate:                          {},
-		godo.DropletOneMinuteLoadAverage:                   {},
-		godo.DropletFiveMinuteLoadAverage:                  {},
-		godo.DropletFifteenMinuteLoadAverage:               {},
-		godo.DropletPublicOutboundBandwidthRate:            {},
-		godo.DbaasFifteenMinuteLoadAverage:                 {},
-		godo.DbaasMemoryUtilizationPercent:                 {},
-		godo.DbaasDiskUtilizationPercent:                   {},
-		godo.DbaasCPUUtilizationPercent:                    {},
-		godo.LoadBalancerCPUUtilizationPercent:             {},
-		godo.LoadBalancerDropletHealth:                     {},
-		godo.LoadBalancerTLSUtilizationPercent:             {},
-		godo.LoadBalancerConnectionUtilizationPercent:      {},
-		godo.LoadBalancerIncreaseInHTTPErrorRatePercentage: {},
-		godo.LoadBalancerIncreaseInHTTPErrorRateCount:      {},
-		godo.LoadBalancerHighHttpResponseTime:              {},
-		godo.LoadBalancerHighHttpResponseTime50P:           {},
-		godo.LoadBalancerHighHttpResponseTime95P:           {},
-		godo.LoadBalancerHighHttpResponseTime99P:           {},
+		godo.DropletCPUUtilizationPercent:                     {},
+		godo.DropletMemoryUtilizationPercent:                  {},
+		godo.DropletDiskUtilizationPercent:                    {},
+		godo.DropletDiskReadRate:                              {},
+		godo.DropletDiskWriteRate:                             {},
+		godo.DropletOneMinuteLoadAverage:                      {},
+		godo.DropletFiveMinuteLoadAverage:                     {},
+		godo.DropletFifteenMinuteLoadAverage:                  {},
+		godo.DropletPublicOutboundBandwidthRate:               {},
+		godo.DbaasFifteenMinuteLoadAverage:                    {},
+		godo.DbaasMemoryUtilizationPercent:                    {},
+		godo.DbaasDiskUtilizationPercent:                      {},
+		godo.DbaasCPUUtilizationPercent:                       {},
+		godo.LoadBalancerCPUUtilizationPercent:                {},
+		godo.LoadBalancerDropletHealth:                        {},
+		godo.LoadBalancerTLSUtilizationPercent:                {},
+		godo.LoadBalancerConnectionUtilizationPercent:         {},
+		godo.LoadBalancerIncreaseInHTTPErrorRateCount4xx:      {},
+		godo.LoadBalancerIncreaseInHTTPErrorRateCount5xx:      {},
+		godo.LoadBalancerIncreaseInHTTPErrorRatePercentage4xx: {},
+		godo.LoadBalancerIncreaseInHTTPErrorRatePercentage5xx: {},
```

Comment:
The LB HTTP alerts were split into 4XX and 5XX alerts

Reply:
Thanks for adding this 🙏